### ISSUE TITLE
[ALM Accelerator - BUG]: ALM Accelerator Cloud flows not enabled after updating to latest release 

### ISSUE THREAD

**Issue by kaat03** [microsoft/coe-starter-kit#5458]
### Describe the issue

After experiencing issues with the Custom Connector, we've updated the solution to the latest release and followed the guide to reflect the changes related to correct Auth Type and Identity Provider.  Prior to that, we have been running the ALM Accelerator without this issue since last year.

Now we cannot see any solutions in the Canvas App. We first got the following error after opening Canvas App:

`GetEnvironmentSolutions.Run failed: { "error": { "code": "WorkflowTriggerIsNotEnabled", "message": "Could not execute workflow '***' trigger 'manual' with state 'Disabled': trigger is not enabled." } }`

After activating the flow manually the following error appears after opening the Canvas App:

`GetEnvironmentSolutions.Run failed: _scrubbedSensitiveData_`

Further inquiry shows that there are several Cloud Flows in the managed solution that haven't been activated:

![image](https://user-images.githubusercontent.com/117084834/236851050-8faf486c-9c01-4276-8da1-58ccc44c1634.png)

On a related note, there seems to be an issue with the Microsoft Dataverse (legacy) connection, which is used by some of the inactive flows.

### Expected Behavior

The Canvas App shows all solutions.

### What component are you experiencing the issue with?

ALM Accelerator Canvas App

### What solution version are you using?

1.0.20230404.2

**Comment by briangre** [comment]
ever get a response on this? I started seeing this error returned from the custom dev ops connector:

Error running pipeline: CustomAzureDevOps.CreatePullRequest failed: _scrubbedSensitiveData_

however searching for hints are only returning confirmation of random outages, but no real clues as to what..  🤷

**Comment by mikefactorial** [comment]
@briangre The _scrubbedSensitiveData_ message has just recently started cropping up in the app. There appears to be a change in the platform that is causing this. Will work with the product team to figure out why it is scrubbing otherwise innocuous error messages. For the time being, however, the most common reason for CustomAzureDevOps.CreatePullRequest failed is because there is already an active PR in Azure DevOps for the same source and target branch.

**Comment by mikefactorial** [comment]
@kaat03 does activating the flows fix your issue?

**Comment by briangre** [comment]
> @briangre the two aren't related. The _scrubbedSensitiveData_ message has just recently started cropping up in the app. There appears to be a change in the platform that is causing this. Will work with the product team to figure out why it is scrubbing otherwise innocuous error messages. For the time being, however, the most common reason for CustomAzureDevOps.CreatePullRequest failed is because there is already an active PR in Azure DevOps for the same source and target branch.

Hey Mike!

I actually found the source of my problem, but it might be worth us getting together on. Using the Canvas App monitoring tool I was able to see that the call to CreatePullRequest was using the same source and target branches. Seemed like maybe an issue in the UI and variables, but not sure.

**Comment by kaat03** [comment]
@mikefactorial We were encountering different kinds of issues trying to activate each flow manually. Unfortunately there wasn't much time to spend on investigating the individual errors and eventually we decided to go with a complete reinstall of the solution. Using all the same versions for the clean install as in the update, which we tried first, went smoothly, without any errors. So something must've gone wrong with the updating to the newest release.
